contrib/opencensus-ext-azure/opencensus/ext/azure/common/utils.py

```diff
@@ -65,3 +66,23 @@ def timestamp_to_iso_str(timestamp):
 
 def url_to_dependency_name(url):
     return urlparse(url).netloc
+
+
+# Validate UUID format
+# Specs taken from https://tools.ietf.org/html/rfc4122
+pattern = re.compile('[0-9a-f]{8}-'
+                     '[0-9a-f]{4}-'
+                     '[1-5][0-9a-f]{3}-'
+                     '[89ab][0-9a-f]{3}-'
+                     '[0-9a-f]{12}')
+
+
+def validate_key(instrumentation_key):
+    if not instrumentation_key:
+        raise ValueError("Instrumentation key cannot be none or empty.")
+    # re.fullmatch not available for python2
+    # We use re.match, we matches from the beginning, and then do a length
```

Comment:
```suggestion
    # Use re.match instead: match from the beginning, and then do a length
```

Reply:
Changed the regex to match full string so comment is not needed.